### ISSUE TITLE
Image tests fail when running tests for that package

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#8837]
## 📝 Provide detailed reproduction steps (if any)

1.  `yarn test --files="image"`

```
FAILED TESTS:
  ImageResizeHandles
    command
      ✖ disables the resizer if the command is disabled
        Chrome 87.0.4280.141 (Mac OS 10.15.7)
      TypeError: Cannot read property 'isEnabled' of undefined
          at Context.eval (webpack:///./packages/ckeditor5-image/tests/imageresize/imageresizehandles.js?:119:20)

      ✖ the resizer is disabled from the beginning when the command is disabled when the image is inserted
        Chrome 87.0.4280.141 (Mac OS 10.15.7)
      TypeError: Cannot read property 'isEnabled' of undefined
          at Context.eval (webpack:///./packages/ckeditor5-image/tests/imageresize/imageresizehandles.js?:144:20)

    undo integration
      ✖ has correct border size after undo
        Chrome 87.0.4280.141 (Mac OS 10.15.7)
      AssertionError: expected '' to equal '120px'

      + expected - actual

      +120px

    at Context.eval (webpack:///./packages/ckeditor5-image/tests/imageresize/imageresizehandles.js?:212:45)


    widget toolbar integration
      ✖ visibility after the resize was canceled
        Chrome 87.0.4280.141 (Mac OS 10.15.7)
      TypeError: Cannot read property 'cancel' of undefined
          at Context.eval (webpack:///./packages/ckeditor5-image/tests/imageresize/imageresizehandles.js?:449:12)

Error: Karma finished with "1" code.
```

## 📃 Other details

I tried bisecting twice and each time I ended up in a different place so it could be there's something random going on.

Fun fact: CI passes https://travis-ci.org/github/ckeditor/ckeditor5/builds/754443472.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by oleq** [comment]
Another fun fact: `yarn test --files="image/imageresize/imageresizehandles.js"` gives 

```
FAILED TESTS:
  ImageResizeHandles
    undo integration
      ✖ has correct border size after undo
        Chrome 87.0.4280.141 (Mac OS 10.15.7)
      AssertionError: expected '100px' to equal '120px'

      + expected - actual

      -100px
      +120px

    at Context.eval (webpack:///./packages/ckeditor5-image/tests/imageresize/imageresizehandles.js?:212:45)
```

but `yarn test --files="image"` with `describe.only( 'ImageResizeHandles', () => {` gives no errors. Something's really weird.

**Comment by oleq** [comment]
Closed in 772314a.
